### DISCUSSION TITLE
Update the submission count for a single user?

### DISCUSSION THREAD

**Discussion by johann-petrak** [codalab/codabench#1506] [Q&A]
We had a user report that his submission took over 30 minutes (we only have small submission files in our competition this is usually done in a few seconds) and that he cancelled the submission, but apparently that submission was still deducted from the possible submission count. Is it possible to reset the submission count for a single user?

**Comment by ihsaan-ullah** [comment]
That submission may have taken more time while waiting in the queue. You cannot change the limit for a single user but if you delete that submission that would remove the submission from the count (if it is counted)
